recipes/beyla-golden-signals/beyla-daemonset.yaml

```diff
@@ -58,20 +58,6 @@ spec:
           env:
             - name: BEYLA_CONFIG_PATH
               value: "/config/beyla-config.yml"
-            - name: OTEL_EXPORTER_OTLP_METRICS_ENDPOINT
-              value: "http://otel-collector:4317"
-            - name: BEYLA_KUBE_METADATA_ENABLE
-              value: "true"
-            - name: BEYLA_SKIP_GO_SPECIFIC_TRACERS
-              value: "true"
-            - name: BEYLA_METRICS_INTERVAL
-              value: "30s"
-            - name: BEYLA_METRICS_REPORT_CACHE_LEN
-              value: "256"
-            - name: BEYLA_TRACES_REPORT_CACHE_LEN
-              value: "256"
```

Comment:
No longer needed as https://github.com/grafana/beyla/issues/496#issuecomment-1908771052 is resolved. Confirmed it's working fine with manual testing after ~1hr 
![image](https://github.com/GoogleCloudPlatform/opentelemetry-operator-sample/assets/1510004/894b7a5a-89bd-4d80-97fc-92c5b6d4062e)